operator/main.py

```diff
@@ -745,6 +745,11 @@ def get_num_pvs(storage_info_data):
         return pv_count
 
     except CommandError as msg:
+        # If storage is created but no PV is carved then pv_stats table is not
+        # created in SQLITE3
+        if msg.stderr.find("no such table") != 1:
```

Comment:
`!= -1`

Reply:
ack, forgot to update important piece while looking into pylint. my bad 😥